sale_order_product_recommendation/wizards/sale_order_recommendation.py

```diff
@@ -217,9 +213,8 @@ class SaleOrderRecommendationLine(models.TransientModel):
     @api.depends("sale_line_id.product_uom", "product_id.uom_id")
     def _compute_sale_uom_id(self):
         for record in self:
-            record.sale_uom_id = (
-                record.sale_line_id.product_uom or record.product_id.uom_id or False
-            )
+            if record.sale_line_id:
```

Comment:
I don't think you should do this if. Anyways, wasn't this done on previous version and you just need to cherry-pick the final solution?

Reply:
@willytal ask to include this PR: https://github.com/OCA/sale-workflow/pull/2065
but it is still open.

I incoporate the code on version 15. 

I couldn't cherryp-pick anything because it does not exists yet.

Reply:
`if` condition's been left as is.